### PR TITLE
Added support for hexadecimal output for OTP generation

### PR DESCRIPTION
**Pull request by ncthompson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,21 +4,20 @@
...
 ## Installation
 
 ```
-$ go get github.com/xlzd/gotp
+$ go get github.com/diebietse/gotp
 ```
 
 ## Usage
 
-Check API docs at https://godoc.org/github.com/xlzd/gotp
```
Comment (jedrivisser) [inline README.md]: A reference to this in some form might still be useful? or maybe ever host your own on godoc if it is easy?

File: README.md [hunk 2]
```diff
@@ -98,12 +109,11 @@ func main() {
...
-[license-badge]:   https://img.shields.io/badge/license-MIT-000000.svg
-[RFC 4226]: https://tools.ietf.org/html/rfc4226 "RFC 4226"
-[RFC 6238]: https://tools.ietf.org/html/rfc6238 "RFC 6238"
-[PyOTP]: https://github.com/pyotp/pyotp
-[License]: https://github.com/xlzd/gotp/blob/master/LICENSE
+[build-status]: https://travis-ci.org/diebietse/gotp.svg?branch=master
+[license-badge]: https://img.shields.io/badge/license-MIT-000000.svg
+[rfc 4226]: https://tools.ietf.org/html/rfc4226 "RFC 4226"
+[rfc 6238]: https://tools.ietf.org/html/rfc6238 "RFC 6238"
+[pyotp]: https://github.com/pyotp/pyotp
```
Comment (jedrivisser) [inline README.md]: py-otp

File: README.md [hunk 3]
```diff
@@ -4,21 +4,20 @@
 
 GOTP is a Golang package for generating and verifying one-time passwords. It can be used to implement two-factor (2FA) or multi-factor (MFA) authentication methods in anywhere that requires users to log in.
 
-Open MFA standards are defined in [RFC 4226][RFC 4226] (HOTP: An HMAC-Based One-Time Password Algorithm) and in [RFC 6238][RFC 6238] (TOTP: Time-Based One-Time Password Algorithm). GOTP implements server-side support for both of these standards.
+Open MFA standards are defined in [RFC 4226][rfc 4226] (HOTP: An HMAC-Based One-Time Password Algorithm) and in [RFC 6238][rfc 6238] (TOTP: Time-Based One-Time Password Algorithm). GOTP implements server-side support for both of these standards.
 
-GOTP was inspired by [PyOTP][PyOTP].
+GOTP was inspired by [PyOTP][pyotp].
```
[the remark this answers is not in the document]
Comment (ncthompson) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (ncthompson) [reply README.md]: Linked to https://godoc.org/github.com/diebietse/gotp

**Pull request merged**